packages/forms/src/directives/radio_control_value_accessor.ts

```diff
@@ -154,10 +157,13 @@ export class RadioControlValueAccessor extends BuiltInControlValueAccessor imple
    */
   @Input() value: any;
 
+  private callSetDisabledState?: SetDisabledStateOption;
```

Comment:
nit: we can use `inject` here (which should also derive the right type for the field):
```suggestion
  private callSetDisabledState = inject(CALL_SET_DISABLED_STATE, {optional: true});
```